javabridge/wrappers.py

```diff
@@ -158,6 +158,12 @@ def __repr__(self):
     
     def __str__(self):
         return J.to_string(self.o)
+    
+    def __int__(self):
+        return self.intValue()
```

Comment:
Your example gives 'False'. Conversion to int is not implicit.

`int(x)` gives `AttributeError`.